shared/constants/plan-billing.js

```diff
@@ -154,3 +157,33 @@ export const updateBillingAndQuota = 'plan-billing:updateBillingAndQuota'
 export type UpdateBillingAndQuota = TypedAction<'plan-billing:updateBillingAndQuota', BillingAndQuota, {error: any}>
 
 export type Actions = UpdateBilling | FetchBillingAndQuota | FetchBillingOverview | UpdateAvailablePlans | UpdateBillingAndQuota
+
+export function planToStars (plan: string): number {
+  return {
+    'Basic': 1,
+    'Gold': 3,
+    'Friend': 5,
+  }[plan] || 0
+}
+
+export type ChangeType = 'change' | 'upgrade' | 'downgrade'
+
+export function comparePlans (from: AvailablePlan, to: AvailablePlan): ChangeType {
```

Comment:
we changed the logic from what zeplin has. If you're free and you change its an upgrade. If you pay and you go free its a downgrade. If you pay and go to a different pay its a change